test-framework/junit5/src/main/java/io/quarkus/test/junit/nativeimage/ClassInclusionReport.java

```diff
@@ -19,7 +19,7 @@
 
     private final Set<String> includedClasses;
 
-    private ClassInclusionReport(Set<String> strings, Path usedClassesReport) {
+    private ClassInclusionReport(Set<String> strings) {
```

Comment:
ah I see this wasn't complete, but I actually meant to store the `usedClassedReport` so to not access filesystem multiple times. Wouldn't it be better?

Reply:
Sure yeah, I'll do it :)

Reply:
Done